packages/anoncreds/tests/legacyAnonCredsSetup.ts

```diff
@@ -158,14 +161,14 @@ export const getAskarAnonCredsIndyModules = ({
     anoncreds: new AnonCredsModule({
       registries: [new IndyVdrAnonCredsRegistry()],
     }),
-    anoncredsRs: new AnonCredsRsModule(),
-    indyVdr: new IndyVdrModule({
```

Comment:
`indyNetworkConfig` is now unused. I think we can remove it if we're using `indyVdrModuleConfig`